plugins/api/solidfire-sioc/pom.xml

```diff
@@ -0,0 +1,47 @@
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one
+  or more contributor license agreements. See the NOTICE file
+  distributed with this work for additional information
+  regarding copyright ownership. The ASF licenses this file
+  to you under the Apache License, Version 2.0 (the
+  "License"); you may not use this file except in compliance
+  with the License. You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing,
+  software distributed under the License is distributed on an
+  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  KIND, either express or implied. See the License for the
+  specific language governing permissions and limitations
+  under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <artifactId>cloud-plugin-api-solidfire-sioc</artifactId>
+  <name>Apache CloudStack Plugin - API SolidFire (SIOC)</name>
```

Comment:
@mike-tutkowski what is SIOC, can this be under `plugins/solidfire/...` than under `plugin/api`?

Reply:
@rhtyd From what I recall, I have enabled this to only run with the -Dnoredist flag. It is an API plug-in, so I just placed it with the other API plug-ins that I found. I'm fine with moving it, though, if you think that's the better way to go. Let me know. Thanks!

Reply:
@mike-tutkowski I think `plugins/api` is for hosting cloudstack's API related plugin such as rate limiting plugin and the api discovery plugin. Perhaps you can consider moving this to `plugin/integrations` (if it's related to integration to external system/service sioc?) or `plugins/storage` or perhaps a more appropriate location?